src/common/config_cacher.h

```diff
@@ -21,12 +21,12 @@
 template <typename ValueT>
 class md_config_cacher_t : public md_config_obs_t {
   ConfigProxy& conf;
+  const char* keys[2];
   const char* const option_name;
```

Comment:
Do we really need `option_name` after introducing `keys`?

Reply:
Good catch. Fixed.